04_setup_ironic.sh

```diff
@@ -44,26 +44,26 @@ sudo podman pod create -n ironic-pod
 sudo podman run -d --net host --privileged --name dnsmasq  --pod ironic-pod \
      -v $IRONIC_DATA_DIR:/shared --entrypoint /bin/rundnsmasq ${IRONIC_IMAGE}
 
-sudo podman run -d --net host --privileged --name httpd --pod ironic-pod \
```

Comment:
I don't think this will work on the pods which need to bind a port on the host network

Reply:
It worked locally, I only removed --privileged, I think "--net host" is enough

Reply:
Ack - that's weird because I thought I tried it and although the container started e.g httpd was not accessible - will re-try.

Reply:
> Ack - that's weird because I thought I tried it and although the container started e.g httpd was not accessible - will re-try.

Ahh, this could be a iptables thing, in my case I would have had them from a previous install, I'll see if I can find out

Reply:
Yup, we need "--privileged" in order to run the iptables commands inside the containers, but it appears to work anyways because we are also setting the same rules in 02_configure_host.sh. We need to remove them in one of these two places.

Reply:
I think the idea was to remove the rules from dev-scripts, then the containers can set the necessary rules wherever we run them, and I know we need to open the ports running as a staticpod on RHCOS, I assume the same is needed when running on the cluster, since we're using net=host @imain ?